### PR TITLE
Adds missing abs to lcm

### PR DESCRIPTION
**Pull request by mruberry**
lcm was missing an abs. This adds it plus extends the test for NumPy compliance. Also includes a few doc fixes.

### CODE REVIEW

File: torch/_torch_docs.py [hunk 1]
```diff
@@ -2915,14 +2919,16 @@ def merge_dicts(*dicts):
 
 add_docstr(torch.isnan,
            r"""
+isnan(input) -> Tensor
+
 Returns a new tensor with boolean elements representing if each element is `NaN` or not.
```
Comment (ngimel) [inline torch/_torch_docs.py:2924]: each element of input

File: torch/_torch_docs.py [hunk 2]
```diff
@@ -2915,14 +2919,16 @@ def merge_dicts(*dicts):
...
 Complex values are considered `NaN` when either their real and/or imaginary part is NaN.
 
 Arguments:
-    input (Tensor): A tensor to check
+    {input}
```
Comment (ngimel) [inline torch/_torch_docs.py:2928]: you are removing comments that input is a Tensor everywhere - is it universally assumed?

**Review by ngimel — APPROVED** [review]

File: torch/_torch_docs.py (hunk 1 above)
Comment (mruberry) [reply torch/_torch_docs.py:2924]: Good idea

File: torch/_torch_docs.py (hunk 2 above)
Comment (mruberry) [reply torch/_torch_docs.py:2928]: You can see the autogenerated text and formatting for input styled like this in the doc for lgamma, for example: https://pytorch.org/docs/master/generated/torch.lgamma.html?highlight=lgamma#torch.lgamma. So yes, it is assumed.

**Pull request closed without merging**